packages/ember-metal/lib/computed_macros.js

```diff
@@ -21,10 +21,10 @@ function getProperties(self, propertyNames) {
 }
 
 function generateComputedWithProperties(macro) {
-  return function() {
+  return () => {
     var properties = a_slice.call(arguments);
 
-    var computedFunc = computed(function() {
+    var computedFunc = computed(() => {
```

Comment:
We should not use arrow function here - we don't want to change `this` in computed macros (it refers to the object the computed is used in). Many examples of this in this file - I won't comment on all.